### DISCUSSION TITLE
〈中華電總座開講〉元宇宙有四大關鍵因素

### DISCUSSION THREAD

**Discussion by chun1203** [1101-GS4719-C/Python-Community-GS4719-C#272] [科技新知]
中華電 (2412-TW) 總經理郭水義今 (5) 日表示，目前元宇宙有四大關鍵元素，包括沉浸式體驗、虛擬化分身、開放式創造、穩定化系統，其中，中華電已打造多個沉浸式體驗的娛樂應用。
元宇宙的四大關鍵元素也需要配套來支撐，以資安最為重要，尤其連結不同宇宙時，所需要的去中心化身分系統、紀錄所有線上行為，以及虛擬世界資料安全性與隱私權問題，其次包含 5G 網路、AI、AR、VR、雲、區塊鏈、終端、平台與內容等。
[https://news.cnyes.com/news/id/4796945](url)

**Comment by 20010906** [comment]
一直出現404的畫面，看不到內容...

**Comment by zixunn** [comment]
好期待將來沉浸式體驗可以變得普及，不只應用在娛樂，也能應用在教育或商業等等領域。

**Comment by 109302003** [comment]
最近不只科技，連藝術作品都很看重沈浸式體驗，約莫三年前開始，看了好多藝廊展覽都是沈浸式體驗。

**Comment by ghost** [comment]
說到沉浸式體驗...就不能忘記那個某小說中會在2022年底上市的NERvGear完全潛行(Full Dive)機器

**Comment by YuHengChen54** [comment]
以後真的可以在家就滿足任何娛樂了，感覺除了吃飯睡覺以外，一整天都可以在虛擬世界度過

**Comment by neo910831** [comment]
感覺現在離真正的沉浸式體驗還很遙遠，目前的VR科技還有不少缺陷(3D暈、操作系統、軟體上的生態還不夠成熟...等等)
之前有看到一篇文章探討現在的腦機介面技術與VR的可能性，非侵入式的EEG效率太低沒辦法發展Real-time的功能。
更不用說Neuralink研究的侵入式技術，現在的道德上也讓它很難普及。
